src/veh_interact.cpp

```diff
@@ -1520,6 +1587,113 @@ void veh_interact::display_list(size_t pos, std::vector<vpart_info> list)
     wrefresh (w_list);
 }
 
+/**
+ * Used when installing parts.
+ * Opens up w_details containing info for part currently selected in w_list.
+ */
+void veh_interact::display_details(const vpart_info &part)
+{
+    if( &part == NULL ) return;
+
+    if (w_details == NULL) { // create details window first if required
+
+        if (vertical_menu) { // clear rightmost blocks of w_stats in vertical/hybrid mode to avoid overlap
+            for (size_t i = 0; i < w_stats->height; i++) {
+                mvwhline(w_stats, i, 34, ' ', w_stats->width - 34);
+            }
+            wrefresh(w_stats);
+        }
+
+        // covers right part of w_name and w_stats in vertical/hybrid, lower block of w_stats in horizontal mode
+        const int details_y = vertical_menu ? w_name->y : w_stats->y + 7;
+        const int details_x = vertical_menu ? w_list->x : w_stats->x;
+
+        const int details_h = vertical_menu ? 6 : 7; // 6 lines in vertical/hybrid, 7 lines in horizontal mode
+        const int details_w = w_grid->x + w_grid->width - details_x;
+
+        w_details = newwin(details_h, details_w, details_y, details_x);
+    }
+    else {
+        werase(w_details);
+   }
+
+    wborder(w_details, LINE_XOXO, LINE_XOXO, LINE_OXOX, LINE_OXOX, LINE_OXXO, LINE_OOXX, LINE_XXOO, LINE_XOOX);
+
+    int column_width = w_details->width / 2; // displays data in two columns
+    int col_1 = vertical_menu ? 2 : 1;
+    int col_2 = col_1 + column_width;
+    int line = vertical_menu ? 0 : 0;
+    bool small_mode = column_width < 20 ? true : false;
+
+    // line 0: part name
+    fold_and_print(w_details, line, col_1, w_details->width, c_ltgreen,
+                   part.name);
+
+    // line 1: (column 1) durability   (column 2) damage mod
+    fold_and_print(w_details, line+1, col_1, column_width, c_white,
+                   "%s: <color_ltgray>%d</color>",
+                   small_mode ? _("Dur") : _("Durability"),
+                   part.durability);
+    fold_and_print(w_details, line+1, col_2, column_width, c_white,
+                   "%s: <color_ltgray>%d\%</color>",
```

Comment:
The print functions don't consider '\' to be special and it's removed by the compiler anyway. Use "%%" to print a single '%', see https://github.com/hbnix/Cataclysm-DDA/blob/vehicle-menu-expansion/src/veh_interact.cpp#L1451